pypesto/profile/profile.py

```diff
@@ -104,6 +104,8 @@ def create_next_guess(
             current_profile_,
             problem_,
             global_opt_,
+            min_step_increase_factor_,
+            max_step_reduce_factor_,
```

Comment:
None of them are really. All "hyperparameters" a user can provide are in `ProfileOptions` or`next_guess_method`. Nothing outside of them should be touched. That's why they're not even mentioned in `parameter_profile`.

Specifically, these two are used to control the reduction of max_step_size and increase of min_step_size in case optimization fails. They are specified by the algorithm in `walk_along_profile.py`, never by the user.